common-content/python-sdk.md

```diff
@@ -1,109 +1,16 @@
 ---
 # Page settings
 layout: default
-keywords:
+keywords: staking
 comments: false
 
 # Hero section
-title: The SingularityNET Command Line Tools
-
-# extralink box
-extralink:
-    title: All Docs
-    title_url: '/docs'
-    external_url: false
-    description: Find an overview of our full documentation here.
-
-# Developer Newsletter
-dev_news: true
+title: Staking
```

Comment:
```suggestion
title: python-sdk
```

Reply:
Ignore title, will have a separate PR to remove that